src/DetailsView/actions/details-view-action-message-creator.ts

```diff
@@ -203,7 +204,7 @@ export class DetailsViewActionMessageCreator extends DevToolActionMessageCreator
         test: VisualizationType,
         requirement?: string,
     ): void {
-        const payload: ToggleActionPayload = {
+        const payload: RequirementToggleActionPayload = {
```

Comment:
Why create a new type instead of using `AssessmentToggleActionPayload`?

Reply:
Updated